src/test/ui/arbitrary-self-types-not-object-safe.stderr

```diff
@@ -17,4 +17,5 @@ LL |     let x = Box::new(5usize) as Box<Foo>;
 
 error: aborting due to 2 previous errors
 
-If you want more information on this error, try using "rustc --explain E0038"
+error: For more information about this error, try `rustc --explain E0038`.
```

Comment:
Inconsistent capitalization between this and other diagnostic messages ("<em><strong>F</strong></em>or more information" but "<em><strong>a</strong></em>borting due to"). (The capitalization here was definitely better when we were just printing on its own line, rather than wrapping it in a diagnostic, as now.)

I could imagine a case that "error: for more information ..." is bad semantics (giving the user more information isn't an error!), but I don't feel very strongly about this.